tests/integration/components/o-s-s/attribute-text-test.ts

```diff
@@ -0,0 +1,53 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+import click from '@ember/test-helpers/dom/click';
+
+module('Integration | Component | o-s-s/attribute-text', function (hooks) {
+  setupRenderingTest(hooks);
+
+  module('Default behavior', function () {
+    test('it renders', async function (assert) {
+      await render(hbs`<OSS::Attribute::Text @label="Hello" @value="World" />`);
+      assert.dom('.oss-attribute--text').exists();
+    });
+
+    test('it renders without a value', async function (assert) {
+      await render(hbs`<OSS::Attribute::Text @label="Hello" />`);
+      assert.dom('.oss-attribute--text').exists();
+    });
+
+    test('it renders with a tooltip when specified', async function (assert) {
+      await render(hbs`<OSS::Attribute::Text @label="Hello" @tooltip="Tooltip" />`);
+      assert.dom('.oss-attribute--info').exists();
+    });
```

Comment:
Add also a test for the tooltip with `await assert.tooltip...`. See doc [here](https://storybook.upfluence.co/?path=/docs/helpers-modifiers-modifiers-tooltip-testing--page)